scipy/constants/codata.py

```diff
@@ -1139,6 +1141,359 @@
 Wien frequency displacement law constant               5.878 9238 e10        0.000 0034 e10        Hz K^-1
 Wien wavelength displacement law constant              2.897 7729 e-3        0.000 0017 e-3        m K"""
 
+txt2018 = """\
+alpha particle-electron mass ratio                          7294.299 541 42          0.000 000 24             
+alpha particle mass                                         6.644 657 3357 e-27      0.000 000 0020 e-27      kg
```

Comment:
This one seems to be showing up in the test failure with `ValueError: could not convert string to float: '6.6446573357e-'`. Indeed, it seems the module is not even importable?

Reply:
Well, the 2018 values are shifted to the right within a line compared to the 2014 values. This could be the problem.

I can try to reformat them, so that they will be aligned with the 2014 values, colmunnwise speaking.

Reply:
The problem is the parsing function:
In the 2018 edition the quantity goes from column 0 to 60, the value from column 60 to 85 and the unit from column 111 to 124. This is different from the editions before.

In the end the decision is up to you: either the parsing function has to be adapted to the new edition or I'll reformat the values.